R/Functions_L2_Colony.R

```
@@ -860,6 +870,9 @@ replaceCastePop <- function(x, caste = NULL, p = 1, use = "rand",
     } else if (isMultiColony(x)) {
       nCol <- nColonies(x)
     }
+    if (nCol == 0) {
```

Do we want an error or do we want to allow empty multiColonies? I guess fail early is OK and we force users to not pass empty multiColonies into multiColony functions. All good.